mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.tmpl

```diff
@@ -1410,6 +1410,30 @@ func TestAccContainerCluster_withPrivateClusterConfigBasic(t *testing.T) {
 	})
 }
 
+func TestAccContainerCluster_withPrivateClusterConfigNoCidrBlock(t *testing.T) {
+	t.Parallel()
+
+	clusterName := fmt.Sprintf("tf-test-cluster-%s", acctest.RandString(t, 10))
+	containerNetName := fmt.Sprintf("tf-test-container-net-%s", acctest.RandString(t, 10))
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:	  func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy: testAccCheckContainerClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccContainerCluster_withPrivateClusterConfigNoCidrBlock(containerNetName, clusterName, false),
```

Comment:
odd thing is that, so far, this test (which I think should fail) is passing for me locally even without the code change, even though I can reproduce the issue / fix with a very similar config.

For example, this gives me an error on apply (though not on plan):
```tf
resource "google_container_cluster" "test_private" {
  name                = "test-private"
  location            = "us-central1-a"
  initial_node_count  = 1
  network             = "default"
  subnetwork          = "default"

  private_cluster_config {
    enable_private_nodes = true
  }
  node_config {
    oauth_scopes = [
      "https://www.googleapis.com/auth/cloud-platform",
    ]
  }

  deletion_protection = false
}
```